.github/workflows/build_and_publish.yml

```diff
@@ -38,3 +38,20 @@ jobs:
      with:
        user: __token__
        password: ${{ secrets.pypi_password }}
+
+ docker-image-CI:
+   name: Docker Image CI
+   runs-on: ubuntu-latest
+   steps:
+
+    - name: Check out git repository
+      uses: actions/checkout@v2
+
+    - name: Build and publish to Docker Hub
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: clinicalgenomics/scout
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        tags: "latest,${{ github.event.release.tag_name }}"
```

Comment:
Nice! I will copy this, never really understood how to use both tags

Reply:
Perhaps you need to change also the action source. I'm using one different from yours here.